### PR TITLE
Edit storage for Tutorial

### PR DESCRIPTION
**Pull request by BlazeChron**
Adds tutorial field to addressbook json. Extra class created to support this.
Adjusted relevant cases to support new field.

### CODE REVIEW

File: src/main/java/seedu/address/storage/JsonAdaptedTutorial.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    public JsonAdaptedTutorial(@JsonProperty("name") String name) {
+        this.name = name;
+    }
+    //    TODO
+    //    public Tutorial toModelType() throws IllegalValueException {
+    //        return new Tutorial(this.name);
+    //    }
+
+    @Override
+    public String toString() {
```
Comment (jinhanjinhan) [inline src/main/java/seedu/address/storage/JsonAdaptedTutorial.java:24]: Hey what is this toString() needed for? Is it needed for writing test code next time?

**Review by jinhanjinhan — COMMENTED** [review]
Just one part that I wasn't so sure of, but overall, great job!

File: src/main/java/seedu/address/storage/JsonAdaptedTutorial.java (hunk 1 above)
Comment (BlazeChron) [reply src/main/java/seedu/address/storage/JsonAdaptedTutorial.java:24]: Good catch, it was code for testing the framework.

**Review by amitha2210 — APPROVED** [review]
LGTM

**Comment by BlazeChron** [comment]
Messed up commit history. Please see #110 instead.

**Pull request closed without merging**